documentation/specs/src/interoperability/ethereum-bridge/proofs.md

```diff
@@ -0,0 +1,78 @@
+# Proofs
+
+A proof for the bridge is a quorum of signatures by a valid validator set
+attached to a message understandable to the Ethereum smart contracts. For
+transferring value to Ethereum, a proof is a signed Merkle tree root and
+inclusion proofs of assert transfer messages understandable to the Ethereum
+smart contractions, as described in the section on 
+[batching](transfers_to_ethereum.md/#batching)
+
+A message for transferring value to Ethereum is a `TransferToNamada` 
+instance as described 
+[here](./transfers_to_ethereum.md/#bridge-pool-validity-predicate).
+
+Additionally, when the validator set changes, the smart contracts on
+Ethereum must be updated so that it can continue to recognize valid proofs.
+Since the Ethereum smart contract should  accept any header signed by bridge
+header signed by 2 / 3 of the staking validators, it needs up-to-date
+knowledge of:
+- The current validators' public keys
+- The current stake of each validator
+
+This means the at the end of every Namada epoch, a special transaction must be
+sent to the Ethereum smart contracts detailing the new public keys and stake
+of the new validator set. This message must also be signed by at least 2 / 3
+of the current validators as a "transfer of power".
+
+If vote extensions are available, this signed data can be constructed
+using them. Otherwise, validators must send protocol txs to be included on
+the ledger. Once a quorum exist on chain, they can be aggregated into a
+single message that can be relayed to Ethereum. Signing an
+invalid  validator transition set will be considered a slashable offense.
+
+Due to asynchronicity concerns, this message should be submitted well in
+advance of the actual epoch change, perhaps even at the beginning of each
+new epoch. Bridge headers to ethereum should include the current Namada epoch
+so that the smart contract knows how to verify the headers. In short, there
+is a pipelining mechanism in the smart contract.
+
+Such a message is not prompted by any user transaction and thus will have
+to be carried out by a _bridge relayer_. Once the transfer of power
+message is on chain, any time afterwards a Namada bridge process may take
+it to craft the appropriate message to the Ethereum smart contracts.
+
+The details on bridge relayers are below in the corresponding section.
+
+Signing incorrect headers is considered a slashable offense. Anyone witnessing
+an incorrect header that is signed may submit a complaint (a type of transaction)
+to initiate slashing of the validator who made the signature.
```

Comment:
```suggestion
Signing incorrect validator sets is considered a slashable offense. Anyone witnessing
an incorrect validator set that is signed may submit a complaint (a type of transaction)
to initiate slashing of the validator who made the signature.
```

Reply:
It's not just validator set updates. It is also just normal transfers.

Reply:
Sorry should have been clearer here, it's the term "header" I thought we should change (thought it was a holdover usage from an older version of the spec)